README.md

```diff
@@ -91,73 +89,25 @@ func main() {
 		// do something
 	}
 
-	// columns you want to exclude from Insert, specify as an argument
+	// Columns you want to exclude from Insert, specify as an argument
 	err = gormbulk.BulkInsert(db, insertRecords, 3000, "Email")
 	if err != nil {
 		// do something
 	}
-}
-```
-
-### BulkInsertWithAssigningIDs
-
-```go
-package main
-
-import (
-	"fmt"
-	"time"
-
-	"github.com/jinzhu/gorm"
-	_ "github.com/jinzhu/gorm/dialects/postgres"
-	gormbulk "github.com/t-tiger/gorm-bulk-insert/v2"
-)
-
-type fakeTable struct {
-	IdPK      uint      `gorm:"primary_key"`
-	CreatedAt time.Time `gorm:"default:now()"`
-	Data      string
-}
-
-func main() {
-	db, err := gorm.Open("postgres", "host=localhost port=5432 user=cloudwalker dbname=cloudwalker password=cloudwalker sslmode=disable")
-	if err != nil {
-		panic(err)
-	}
-	defer db.Close()
-	db.SingularTable(true)
-
-	if err := db.AutoMigrate(fakeTable{}).Error; err != nil {
-		panic(err)
-	}
 
-	models := []interface{}{
-		fakeTable{Data: "aaa"},
-		fakeTable{Data: "bbb"},
-		fakeTable{Data: "ccc"},
+	// Fetch returning values
+	dbForReturning := db.Set("gorm:insert_option", "RETURNING id, name, created_at")
+	var returned []struct {
+		ID        int
+		Name	  string
+		CreatedAt time.Time
 	}
```

Comment:
if i only need `ID`, i would define
```
var returned []struct{
    ID uint
}
```
scan it from result then
```
var ids []uint
for _,ret := range returned{
    ids = append(ids, ret.ID)
}
```
it's odd...
its nessessary to support *[]uint or *[]string, i suggest.